src/components/RichContenteditable/RichContenteditable.vue

```diff
@@ -181,6 +218,31 @@ export default {
 			return !!navigator.userAgent.match(/firefox/i)
 		},
 
+		/**
+		 * Is the current value over maxlength?
+		 * @returns {boolean}
+		 */
+		isOverMaxlength() {
+			if (this.isEmptyValue || !this.maxlength) {
+				return false
+			}
+			return this.localValue.length > this.maxlength
```

Comment:
what about multibyte? do we want to allow 100x `!` just like 100x `🧰`?

Reply:
I guess this will usually be bound to the database (thus us requiring max-lengths).
How would the db count?

Reply:
mb_strlen
https://github.com/nextcloud/server/blob/0c7c358d134e4f12b18364f23e32b991176dabc1/apps/user_status/lib/Service/StatusService.php#L253

Reply:
https://www.npmjs.com/package/string-length